src/fragments/lib-v1/storage/js/copy.mdx

```diff
@@ -34,19 +31,19 @@ You can copy a file from the specified key to another key within the same [File
 ```javascript
 import { copy, list } from 'aws-amplify/storage';
 
-console.log(await list({ path : 'copied/' })); // []
+console.log(await list({ prefix : 'copied/' })); // []
 
 // Copies 'existing/srcKey' to 'copied/destKey' within the 'guest' access level
 const copied = await copy({ { source: { key : 'existing/srcKey' } }, { destination: { key : 'copied/destKey' } } });
 
 // There should now be a new file with key "copied/destKey"
-console.log(list({ path : 'copied/' })); // [ { ..., key: 'copied/destKey' } ]
+console.log(list({ prefix : 'copied/' })); // [ { ..., key: 'copied/destKey' } ]
 console.log(copied); // { key: 'copied/destKey' }
 ```
 
 ## Copy files across access levels
 
-To copy file across different access levels, you'll need to explicitly provide the source and destination access levels:
+To copy a file across different access levels, you'll need to explicitly provide the access level if it's 'private' or 'protected' (Defaults to 'guest').
```

Comment:
can you please mention that we have to provide accessLevel on both the source and destination if they need to copy across different levels

Reply:
When coping across access levels I think they need to explicitly provide `accessLevels` only if it's private or protected. I think we would always default to guest in source/destination if `accessLevels` isn't provided

guest (accessLevels isn't required) -> private/protected (accessLevels required)
private/protected (accessLevels required) -> guest (accessLevels isn't required)

Reply:
Yes. But can you please mention to provide accessLevel on both `source` and `destination` if not guest then.

Reply:
Ahh i see the suggestion was to add the key words `source` and `destination`. Sorry i thought you were suggesting to mention accessLevels for guest as well. 

This makes sense updated to 
`To copy a file across different access levels, you'll need to explicitly provide the source and destination access levels if it's 'private' or 'protected' (Defaults to 'guest').`